README.md

```diff
@@ -1,5 +1,46 @@
-# memestr
+# Memestr
 
-An open-source project that delivers happiness in form of memes. Try it now.
+🚀 Welcome to Memestr, the ultimate hub for memes, trending content, and all
+things hilariously entertaining! 🎉
 
-https://memestr.app
+Try Now 🎊 https://memestr.app
+
+## What is Memestr?
+
+Memestr is a community where humour takes centre stage. As
+an
+open-source platform, Memestr empowers users to express themselves through the
+art of memes and engage with a diverse community of meme enthusiasts.
+
+## What sets it apart from similar projects?
```

Comment:
should also add that memestr is a media only client, @nemanjaglumac what do you think?

Reply:
Would update.